hack/e2e-test.sh

```diff
@@ -0,0 +1,28 @@
+export KUSTOMIZE=$PWD/bin/kustomize
+export GINKGO=$PWD/bin/ginkgo
+export KIND=$PWD/bin/kind
+function cleanup {
+    if [ $USE_EXISTING_KIND == 'false' ] 
+    then 
+        $KIND delete cluster --name $KIND_CLUSTER_NAME
+    fi
+}
+function startup {
+    if [ $USE_EXISTING_KIND == 'false' ] 
+    then 
+        $KIND create cluster --name $KIND_CLUSTER_NAME --image $E2E_KIND_VERSION
+    fi
+}
+function kind_load {
+    $KIND load docker-image $IMAGE_TAG --name $KIND_CLUSTER_NAME
+}
+function kueue_deploy {
+    cd config/components/manager && $KUSTOMIZE edit set image controller=$IMAGE_TAG
```

Comment:
undoing this should be part of the cleanup too.

Reply:
So I do have this code that I just used from the Makefile.

```
.PHONY: test-e2e
test-e2e: manifests generate fmt vet envtest ginkgo kind-image-build
	E2E_KIND_VERSION=$(E2E_KIND_VERSION) KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) USE_EXISTING_KUBERNETES=$(USE_EXISTING_KUBERNETES) ARTIFACTS=$(ARTIFACTS) IMAGE_TAG=$(IMAGE_TAG) ./hack/e2e-test.sh
	@$(call clean-manifests)
```
This does do the undoing but I realize it could be good to put in the bash script.  But we do use this in our deploy and undeploy tests in the Makefile already.

Reply:
Moved also to our script.